arduino/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.0.0)
+project(comunication VERSION 0.1.0)
+
+# i need that import for the threads to work propraly 
+add_compile_options("-Wall")
+include_directories("./include")
+# also like line 4
```

Comment:
dont tell me like line 4 because if i will add something before line 4 it will no longer be line 4 therefor just duplicate the comment